apps-rendering/src/components/series.tsx

```diff
@@ -66,6 +66,12 @@ const liveLinkStyles = (theme: ArticleTheme): SerializedStyles => css`
 	text-decoration: none;
 `;
 
+const deadBlogLinkStyles = (theme: ArticleTheme): SerializedStyles => css`
+	${headline.xxxsmall({ lineHeight: 'tight', fontWeight: 'bold' })}
+	color: ${getThemeStyles(theme).deadblogKicker};
```

Comment:
Is there an abstraction where we have a function that accepts both`Design` and `Theme` (or is that part of an ongoing wider discussion)? Agree that here it makes sense to duplicate the linkStyles function for deadBlogs, but wondering how this might scale for other variations between `Design`.